tests/kernel/run_ltp.pm

```diff
@@ -405,7 +411,21 @@ This overrides LTP_COMMAND_PATTERN.
 
 =head2 LTP_TIMEOUT
 
-The time in seconds which each test command has to run.
+The time in seconds which each test command has to run. The default is 900.
+
+=head2 LTP_TIMEOUT_test (e.g.  LTP_TIMEOUT_zram01)
+
+Test specific timeout in sec (overrides LTP_TIMEOUT).
+
+=head2 LTP_TIMEOUT_MUL
+
+Multiplies the timeout. Also exported as environment variable for LTP
+(originally LTP variable).
+
+=head2 LTP_TIMEOUT_MUL_arch (e.g. LTP_TIMEOUT_MUL_aarch64)
```

Comment:
I think all should be upper case.

Reply:
Sure, I'll change it. BTW it'd be interesting to see that documented.

Reply:
You both were right, variable is not set => need to be upper case.